worknenjoy/gitpay
# Issue#848 - Bug on send solution screen

## Description

> Removing unnecessary throwing error that was causing a misbehavior of the task solution dialog.

## Changes

- Task Solution Dialog (frontend)
- Task Solution Get (backend)

## Issue

> Closes #848 

## Impacted Area

> Task solution dialog

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- Create/Import Task
- With a different user, open the task solution dialog and verify if will throw any error.

## Before
> 
![image](https://user-images.githubusercontent.com/21283066/129647319-995505d1-e236-4a7f-846e-4f3ae6d58e51.png)


## After
> 
![image](https://user-images.githubusercontent.com/21283066/129647364-c3ef6eab-7231-4e72-a6c9-2c739fb45ac4.png)
